index.css

```diff
@@ -0,0 +1,569 @@
+    height: 123px;
+    font-family: inherit;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    position: relative;
+    left: 170px;
+    top: 110px;
+    overflow: hidden;
+    color: #f4f5f7;
+  }
+
+  .muñeco1 {
+    width: 673px;
+    position: absolute;
+    left: 760px;
+  }
+
+  .about {
+    width: 566px;
+    height: 190px;
+    font-family: crete round, sans-serif;
+    font-size: 50px;
+    position: relative;
+    left: 160px;
+    top: 160px;
+    color: #ff4500;
+  }
+
+  .lenguage {
+    position: absolute;
+    bottom: 190px;
+    width: 114px;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 22px;
+    line-height: 43px;
+    color: #fff;
+  }
+
+  .main2 {
+    position: relative;
+    background-color: #1c1a19;
+    width: 1440px;
+    height: 1150px;
+  }
+
+  .letras2 {
+    display: flex;
+    align-items: center;
+    padding-left: 10px;
+    width: 226px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    line-height: 44px;
+  }
+
+  .rectangle {
+    display: flex;
+    align-items: center;
+    width: 90px;
+    height: 40px;
+  }
+
+  .row {
+    position: relative;
+    bottom: 160px;
+  }
+
+  .letras4 {
+    width: 114px;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 22px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .get {
+    position: relative;
+    top: 150px;
+    display: flex;
+    align-items: center;
+    left: 170px;
+    width: 161px;
+    height: 48px;
+    background-color: #f55800;
+  }
+
+  .bottong {
+    width: 137px;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .get:hover {
+    background-color: #ecd096;
+  }
+
+  footer {
+    width: 1440px;
+    height: 500px;
+    background-color: #fff;
+  }
+
+  .always h3 {
+    width: 500px;
+    height: 256px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    position: relative;
+    left: 120px;
+    top: 60px;
+    color: #262626;
+    display: inline-block;
+  }
+
+  .formato {
+    float: right;
+    position: relative;
+    right: 250px;
+    top: 90px;
+  }
+
+  .lastname {
+    position: relative;
+    left: 280px;
+    bottom: 23px;
+  }
+
+  #msg {
+    position: relative;
+    top: 30px;
+    background-color: #e2c2dd;
+    width: 500px;
+    height: 80px;
+  }
+
+  .palet2 {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    position: absolute;
+    left: 600px;
+  }
+
+  .palet2 li {
+    margin-left: 10px;
+  }
+
+  .line3 {
+    position: relative;
+    width: 1430px;
+    height: 10px;
+    top: 110px;
+    background-color: red;
```

Comment:
- [ ] Kindly make the `height: 10px;` smaller to be like the Figma design. Like 1px or 2px.